agent/statemanager/state_manager_test.go

```diff
@@ -104,8 +104,8 @@ func TestLoadsV13DataCorrectly(t *testing.T) {
 	err = stateManager.Load()
 	assert.NoError(t, err)
 
-	assert.Equal(t, cluster, "test")
-	assert.True(t, sequenceNumber == 0)
+	assert.Equal(t,  "test", cluster)
+	assert.Equal(t, 0, sequenceNumber)
```

Comment:
You'll need `assert.Equal(t, int64(0), sequenceNumber)` to make this pass.

Reply:
or `EqualValues` ?

Reply:
Yep, that'd work too.